wiki/Client/Interface/Chat_console/Highlight/en.md

```diff
@@ -0,0 +1,19 @@
+---
+tags:
+  - mention
+  - hilight
+---
+
+# Highlight
+
+**Highlight** (sometimes called a **mention**) is a way to draw someone's attention to the chat if they are online. By default, highlight is triggered when a player's username is written out in the chat. The list of additional highlight words may be customized in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
```

Comment:
```suggestion
A **highlight** (commonly abbreviated as **HL**, sometimes called a **mention** or **ping**) is a way to draw someone's attention to the chat if they are online. By default, a highlight is triggered when a player's username is written out in the chat. The list of what words get highlighted may be customised in osu!'s [in-game chat options](/wiki/Client/Options#in-game-chat).
```

Reply:
going to accept it for the sake of simplicity, but words aren't getting highlighted, the whole line does